pennylane/devices/__init__.py

```diff
@@ -37,6 +37,9 @@
     default_qutrit_mixed
     default_clifford
     default_tensor
+    _legacy_device
+    _qubit_device
+    _qutrit_device
```

Comment:
Should these be rendered in the docs? I see value in having the legacy device base classes in the development guide, but I'm not sure if it needs to be exposed to _all_ users. I'm impartial, but just curious.

Reply:
The devices page belongs to the "Internals" section of the docs, so I believe this is okay (?)